### PR TITLE
Update example grain config to match the updated structure

### PR DESCRIPTION
**Pull request by topocount**
Also linted the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,74 +51,92 @@ to master and using the Github Action.
...
+how the grain gets distributed: `immediatePerWeek`, `balancedPerWeek`, and `recentPerWeek`. For info on what each policy does, how to choose the right policy for your community, and how Grain operates in general, see [How Grain Works](https://sourcecred.io/docs/beta/grain).
 
-Below is an example `grain.json` file for a configuration that uses a combination of all three policies. Here we tell SourceCred to distribute 1,000 grain every week, with 25% (250 grain) distributed according to `immediatePerWeek`, 25% (250 grain) distributed according to `balancedPerWeek`, and 50% (500 grain) distributed according to `recentPerWeek`. 
+Below is an example `grain.json` file for a configuration that uses a combination of all three policies. Here we tell SourceCred to distribute 1,000 grain every week, with 25% (250 grain) distributed according to `immediatePerWeek`, 25% (250 grain) distributed according to `balancedPerWeek`, and 50% (500 grain) distributed according to `recentPerWeek`.
```
Comment (hozzjss) [inline README.md:112]: here you could see the old parameters, need changes

File: README.md [hunk 2]
```diff
@@ -50,74 +51,92 @@ to master and using the Github Action.
...
 
-```
+```json
 {
- "immediatePerWeek": 250,
- "balancedPerWeek": 250,
- "recentPerWeek": 500,
- "recentWeeklyDecayRate": 0.5,
  "maxSimultaneousDistributions": 100
+ "allocationPolicies": [
```
Comment (hozzjss) [inline README.md:117]: I would only ask for you to add some docs for the SPECIAL distribution as well but it's all good if it was meant to make these docs correct

File: README.md [hunk 3]
```diff
@@ -50,74 +51,92 @@ to master and using the Github Action.
...
 
 This repo also contains a GitHub action for automatically distributing grain. It will run every Sunday and create a Pull Request
 with the ledger updated with the new grain balances based on the users Cred scores. The amount of grain to get distributed
-every week can be defined in the `config/grain.json` file. 
-
+every week can be defined in the `config/grain.json` file.
 
 There are three different policies that can be used to control
-how the grain gets distributed: `immediatePerWeek`, `balancedPerWeek`, and `recentPerWeek`. For info on what each policy does, how to choose the right policy for your community, and how Grain operates in general, see [How Grain Works](https://sourcecred.io/docs/beta/grain). 
+how the grain gets distributed: `immediatePerWeek`, `balancedPerWeek`, and `recentPerWeek`. For info on what each policy does, how to choose the right policy for your community, and how Grain operates in general, see [How Grain Works](https://sourcecred.io/docs/beta/grain).
```
Comment (hozzjss) [inline README.md:110]: immediatePerWeek, balancedPerWeek, and so on should change

**Review by hozzjss — COMMENTED** [review]
some changes but all in all it's good

File: README.md (hunk 2 above)
Comment (topocount) [reply README.md:117]: I don't think its really advisable to create `SPECIAL` distributions via the grain config file. There is a UI route that is much less clumsy and heavy-handed.

**Review by blueridger — APPROVED** [review]

**Pull request merged**